angular-google-analytics/angular-google-analytics.d.ts

```diff
@@ -0,0 +1,173 @@
+// Type definitions for angular-google-analytics v1.1.0
+// Project: https://github.com/revolunet/angular-google-analytics
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../angularjs/angular.d.ts" />
+
+declare module angular.google.analytics {
+    /**
+     * @summary Interface for {@link AnalysticsProvider}.
+     * @interface
+     */
+    interface IAnalyticsProvider {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.